staff/views/batch.py

```diff
@@ -219,3 +216,14 @@ def new_batch_schedules(batch, batch_schedule_formset):
             )
 
     return new_batch_schedules
+
+def set_up_section(batch, section_number, section_capacity):
+    slack_client = Slack()
+    slack_channel_name = f"{batch.number}-{section_number}"
```

Comment:
Could we double-check the preferred naming with @Nuuggs ?

Reply:
we decided not to create section channels so early. we will create a `<batch_number>-all` at this point channel instead